src/components/HomepageCard/HomepageCard.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./HomepageCard.scss";
+
+const HompageCard = ({ title, link }) => {
+  return (
+    <div className="homepage">
```

Comment:
Can we change this classname to be "homepage-card" and then change the subsequent classes that follow. I just think that "homepage" is very generic